packages/shared/test/fixtures/shared-secret-vectors.json

```diff
@@ -0,0 +1,235 @@
+{
+  "test_vectors": [
```

Comment:
I noticed the fixed nonce, specific shared secret - does it make sense to also have more random values in the test vector?  Or is that something planned for later in the process?

Reply:
The values are randomly generated at the Rust code and fed as constants to the compatibility tests. For values that are randomly generated in this project, there is the unit test in the other file.

Reply:
> The values are randomly generated at the Rust code and fed as constants to the compatibility tests.

Perhaps I'm misunderstanding. The values for the nonce, shared secret don't seem that "random"...? In one case the nonce is just 0s or 1s and the shared secret is just ints 0-31...?

Reply:
Yeah, sorry my bad. I forgot that I updated this in Rust. Yes, the values are fixed. But since the values does not have special numerical meanings, in relation to the algorithm, they are still counted as random (or you may say simi-random).
However, we can generate different values every time. But it will make debugging harder in case of an issue.

Hey @cygnusv, Could you please, provide your input if testing with new random values is needed for this cryptography validation.

Reply:
Would like continue this the conversation about the generated test vectors at https://github.com/nucypher/nucypher-core/pull/105.
Thanks,

Reply:
Sorry for being a bit late to the discussion. I've been following along while working on https://github.com/nucypher/nucypher-core/pull/105 and I changed my mind a few times during this process. The production and consumption of test vectors are a bit more complicated than expected due to how our internal protocol objects work, since in some cases serialization/deserialization is more difficult or not possible without adding code. I'm still iterating a proper method for producing test vectors in my PR, but on the PRs in taco-web we should strive to consume test vectors without resorting to duplicated implementations.

Reply:
With respect to the specific issue discussed here, the test vector ideally should be generated in a 100% deterministic way (i.e. using a seed). However, there's nothing wrong about adding some hard-coded, obviously not random examples, like `[1, 1, 1, .....1]`.